migrations/versions/0299_org_types_table_.py

```diff
@@ -0,0 +1,43 @@
+"""
+
+Revision ID: 0299_org_types_table
+Revises: 0298_add_mou_signed_receipt
+Create Date: 2019-07-10 16:07:22.019759
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = '0299_org_types_table'
+down_revision = '0298_add_mou_signed_receipt'
+
+
+def upgrade():
+    organisation_types_table = op.create_table(
+        'organisation_types',
+        sa.Column('name', sa.String(), nullable=False),
+        sa.PrimaryKeyConstraint('name'),
+        sa.Column('is_crown', sa.Boolean, nullable=True),
+        sa.Column('annual_free_sms_fragment_limit', sa.BigInteger, nullable=False)
+
+    )
+
+    op.bulk_insert(
+        organisation_types_table,
+        [
+            {'name': x, 'is_crown': y, 'annual_free_sms_fragment_limit': z} for x, y, z in [
+                ["Central government", None, 250000],
```

Comment:
Can we store something machine-readable in the database and let the admin app deal with the human readable label? I really don’t want to have to deal with database migrations to change these values. Would suggest:
- `central`
- `local`
- `nhs_central`
- `nhs_local`
- `emergency_services`
- `school_or_college`
- `other`